30.09 praktika/30.09 praktika/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _30._09_praktika
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("Введите длину ребра правильного тетраэдра:");
+            var length = double.Parse(Console.ReadLine());
+            Console.WriteLine("Объем правильного тетраэдра примерно равен " + Math.Round(Math.Sqrt(2) * length * length * length / 12));
+            Console.WriteLine("Площадь поверхности правильного тетраэдра примерно равен " + Math.Round(Math.Sqrt(3) * length * length));
```

Comment:
Подученные величины не нужно было округлять до целых. В задании про округление ничего не сказано.
```suggestion
            Console.WriteLine("Объем правильного тетраэдра примерно равен " + Math.Sqrt(2) * length * length * length / 12);
            Console.WriteLine("Площадь поверхности правильного тетраэдра примерно равен " + Math.Sqrt(3) * length * length);
```